src/state-calculator-form.tsx

```diff
@@ -110,16 +110,22 @@ const renderEmailField = (
     />
     <div className="mt-1 mx-3 text-color-text-secondary text-xsm leading-normal">
       <slot name="email-helper">
-        {msg(
-          'Get updates on incentives, rebates, and more from Rewiring America.',
-        )}{' '}
-        {msg('View our')}{' '}
+        {msg('By sharing your email, you agree to our')}{' '}
```

Comment:
This one should have a description to tell translators what word is going to be next:

```
msg('By sharing your email, you agree to our', { desc: 'will be followed by "terms", as in terms of service' })
```
```